### DISCUSSION TITLE
You need to enable JavaScript to run this app. Chaos Dashboard

### DISCUSSION THREAD

**Discussion by jpucheta** [chaos-mesh/chaos-mesh#3360] [Q&A]
Hello, I'm trying to use Chaos Dashboard from a static URL, but we are getting "You need to enable JavaScript to run this app."  Most of the solution is to enable JavaScript in the Browser or use react or add a proxy to package.json file ...
Anyone faced the same issue? we want to avoid using it with kubectl port-forward  .

![image](https://user-images.githubusercontent.com/91070820/173675928-5d392c41-5adb-4b20-8210-76415975f9f5.png)
![image](https://user-images.githubusercontent.com/91070820/173675999-55f378ae-4708-40cf-8979-04e57ff0328e.png)

**Comment by jpucheta** [comment]
Solved!

**Comment by fangtaozc** [comment]
Hello, could you tell me how it was solved? Would you mind telling me?

**Reply by puchevcp** [reply]
Hi There... In my case we were using Kong as Ingress Controller to expose Chaos Dashboard URL and for some reason we were getting the Java Error, so we tried to expose it using nginx instead and worked fine . It could be an old kong version that we were using but not sure, at least using nginx worked for us.
Hope that helps!

**Reply by xcbeyond** [reply]
> Hi There... In my case we were using Kong as Ingress Controller to expose Chaos Dashboard URL and for some reason we were getting the Java Error, so we tried to expose it using nginx instead and worked fine . It could be an old kong version that we were using but not sure, at least using nginx worked for us. Hope that helps!

@puchevcp I also encountered this problem. I accessed dashboard through nginx ingress. Is there any special configuration for ingress？
